### PR TITLE
Vultr: Introducing vr_plan_facts module

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

This commit introduces a new module called vr_plan_facts.

This module aims to return the list of plan avaiable avaiable to use on
booted servers.

Sample available here:

```
"vultr_plan_facts": [
  {
    "VPSPLANID": "118",
    "available_locations": [
      {
        "DCID": "1",
        "name": "New Jersey"
      }
    ],
    "bandwidth": "40.00",
    "bandwidth_gb": "40960",
    "disk": "110",
    "name": "32768 MB RAM,110 GB SSD,40.00 TB BW",
    "plan_type": "DEDICATED",
    "price_per_month": "240.00",
    "ram": "32768",
    "vcpu_count": "8",
    "windows": false
  }
]
```

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

  - vr_plan_facts

##### ANSIBLE VERSION
  
- devel

##### ADDITIONAL INFORMATION

  - N/A

### CODE REVIEW

File: test/sanity/validate-modules/ignore.txt [hunk 1]
```diff
@@ -520,6 +520,7 @@ lib/ansible/modules/cloud/vultr/vr_dns_record.py E326
 lib/ansible/modules/cloud/vultr/vr_firewall_group.py E324
 lib/ansible/modules/cloud/vultr/vr_firewall_rule.py E323
 lib/ansible/modules/cloud/vultr/vr_firewall_rule.py E324
+lib/ansible/modules/cloud/vultr/vr_plan_facts.py E324
```
Comment (mattclay) [inline test/sanity/validate-modules/ignore.txt]: New ignores should not be added. Instead, the underlying issue should be fixed.

If the issue is in existing common code (docs fragments or module_utils), then the fix should be in a separate PR.

Comment (Spredzy) [reply test/sanity/validate-modules/ignore.txt]: @mattclay agree. I did that while https://github.com/ansible/ansible/pull/42464 gets merged, so the CI feedback is green. Once merged, I'll update this PR removing this line from here.

**Review by resmo — APPROVED** [review]
shipit
